src/main/java/nl/tudelft/contextproject/model/Drawable.java

```diff
@@ -8,7 +8,7 @@
  * Interface representing a Drawable object.
  * This object has a geometry that can be updated.
  */
-public interface Drawable {	
+public interface Drawable extends Collidable {	
```

Comment:
This is a potential code smell, as it implies that every object that we could possibly draw on the screen is solid (non pass through).